src/payment/strategies/stripe-upe/stripe-upe.ts

```diff
@@ -0,0 +1,236 @@
+/**
```

Comment:
i think there is a lot of things in this file that can be reused from `stripev3.ts`